spaces/S000142/properties/P000184.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000142
+property: P000184
+value: true
+refs:
+  - mr: 482697
+    name: Dimension Theory (Engelking)
+---
+
+There is a homeomorphic embedding of $X$ into $\mathbb R^2$.  See sections 1.2.15 and 1.4.6 in {{mr:482697}} for details.
```

Comment:
I don't have my copy handy - is this result explicit there? If so this seems fine; otherwise it's maybe too much original work here..

Reply:
Section 1.4.6 provides an explicit homeomorphic embedding of the space into the unit square (with a picture even).